projects/archiver/src/notifications/device-notifications.ts

```diff
@@ -23,8 +24,10 @@ export const scheduleAppStoreNotification = (
     const { issueDate } = issuePublication
     const { key, name } = issue
 
+    const keyToGUID = uuid.fromString(key)
```

Comment:
I'm pretty sure this won't work as this method assumes that key is a valid string representation of a UUID. I think a better strategy would be to use something like murmurhash4 to hash the key and then 'fit' the resulting string into a UUID format using slice.  i.e.
```
const h = murmurHash3.x64.hash128(key)
const uuid = `${h.slice(0,8)}-${h.slice(8,12)}-${h.slice(12,16)}-${h.slice(16,20)}-${h.slice(20,32)}`
```

Whilst this is a violation of UUIDs - this will pretty much guarantee the required unique property.

Reply:
i am using this lib: `https://www.npmjs.com/package/uuidv4`
it have api to generate uuid form any string it may do something similar as above

i can use your suggestions, but this one works as well